src/test/integration/integration-test.py

```diff
@@ -62,17 +62,17 @@ def checking_spending_limits(self, attributes):
                 self.assertTrue(limit['indexes'])
                 spending_limit.append(limit['spendingLimit'])
         except KeyError as error:
-            logging.debug(f'{error} don\'t exist')
+            logging.debug(f'{error} does\'t exist')
```

Comment:
Since you use the entire `error` object as part of the format string, I'm curious if the substring: `doesn't exist` is needed. What does the error message look like?

Reply:
it would be DEBUG:root:'limits2' does't exist

Reply:
Okay, sounds good to me.

Reply:
If i only log error then it will only display the name of the object that does not exist, should I keep it the way it is for more details?

Reply:
Yeah, more details is better.